pcm-pcie.cpp

```diff
@@ -376,7 +376,7 @@ int main(int argc, char * argv[])
                         cout << "," << ((sample[i].total.ItoM + sample[i].total.RFO)*64ULL);
                     }
                     cout << "	(Total)\n";
-					
```

Comment:
it would be much nicer if all unrelated formatting improvements (incl. dead space elimination) would belong to a different commit.